pyteal/compiler/compiler.py

```diff
@@ -1,4 +1,7 @@
 from typing import List, Tuple, Set, Dict, Optional, cast
+from pyteal.compiler import optimizer
```

Comment:
@algoidurovic Unrelated to the line:  Since we're mostly agreed on the optimization to-be-provided, can you update the user guide?

Nominally, https://pyteal.readthedocs.io/ ought to explain how users can opt-in for optimization.